source/installation/index.html

```diff
@@ -186,7 +113,84 @@ <h3>Install Home Assistant on Raspberry Pi</h3>
       </div>
     </div>
   </div>
+</div>
+
+
+ <h2>Other installation methods</h2>
```

Comment:
```suggestion
 <h2>About installation methods</h2>
```
These are not "other" installation methods than the ones listed above. The Home Assistant Green installation, for example, uses the Home Assistant Operating System.